spec/tapioca/cli_spec.rb

```diff
@@ -0,0 +1,30 @@
+# typed: true
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Tapioca
+  class CliSpec < SpecWithProject
+    describe "Tapioca::Cli" do
+      before do
+        @command_stub = mock(run: nil)
```

Comment:
This is just my preference, but I would avoid the extra dependency and just use `Minitest::Mock` here. I don't know what's the opinion for the rest of the team